app/views/play/level/PlayGameDevLevelView.coffee

```diff
@@ -119,7 +120,7 @@ module.exports = class PlayGameDevLevelView extends RootView
 
   onEditLevelButton: ->
     viewClass = 'views/play/level/PlayLevelView'
-    route = "/play/level/#{@level.get('slug')}"
+    route = "/play/level/#{@level.get('slug')}?course=#{@courseID}&course-instance=#{@courseInstanceID}"
```

Comment:
What if the user isn't plaything though a course?  Seems weird to set these to empty.